CONTRIBUTING.md

```diff
@@ -60,4 +60,4 @@ by two units (that is, four characters).
 | A(x) ->
 >>>>let y = ... in
 >>>>...
-```
+```
```

Comment:
useless modif

Reply:
I don't know why this line is highlighted.

Reply:
Perhaps a slightly different character code. Just copy the original file.